.github/workflows/pr.yml

```diff
@@ -21,10 +25,14 @@ jobs:
         steps:
             - name: Checkout the Git repository
               uses: actions/checkout@v2
-            - name: Setup Toltec environment
+            - name: Setup Python
+              uses: actions/setup-python@v2
+              with:
+                python-version: '3.x'
+            - name: Setup Toltec dependencies
               uses: ./.github/actions/setup
             - name: Build packages
-              run: remote_repo='https://toltec-dev.org/${{ github.base_ref }}' make repo-new
+              run: make repo-new FLAGS='--remote-repo https://toltec-dev.org/${{ github.base_ref }}'
```

Comment:
i know it's not related, but both linus and I have separate repos we use for testing (hosted on different URLs than toltec) - would it be possible to make the URL configurable here? (we can do in a separate commit, but it just came to mind while reviewing)

Reply:
Good point. Would using a secret work for your use case?

Reply:
yes, definitely

Reply:
I don’t think that there’s a way to add a default value to a GitHub Action expression unfortunately. But it should be fairly easy to set that secret in the Toltec org to point to our server.

Reply:
Added a new secret called `REMOTE_HTTP` which parameterizes `https://toltec-dev.org`. Renamed the existing `REMOTE` secret to `REMOTE_SSH` for clarity.